easysid.cpp

```diff
@@ -14,10 +14,29 @@
 #endif
 
 SID sid;
-PcmOutput *pcm;
+PcmOutput *pcm = NULL;
+PcmOutput *wav = NULL;
 
 extern "C" {
 
+EXPORT void set_capture(const char *fn)
+{
+    if (wav != NULL) {
+        delete wav;     // Deleting the existing capture, will close the current file, and start a new
```

Comment:
Fix the comment. It's too long and has repeated words. Prefer comments on their own lines preceding the code in question, unless they are very short.